cdap-common/bin/cdap

```diff
@@ -74,7 +74,7 @@ case ${1} in
   upgrade) shift; cdap_upgrade_tool ${@}; __ret=${?} ;;
   run) shift; cdap_run_class ${@}; __ret=${?} ;;
   sdk) shift; cdap_sdk ${@}; __ret=${?} ;;
-  setup) shift; cdap_setup_tool ${@}; __ret=${?} ;;
+  setup) shift; cdap_hbase_setup ${@}; __ret=${?} ;;
```

Comment:
I was thinking of the argument being more `hbase_setup` than the function name. Sorry that I wasn't clear.

However, this isn't something that Nitin likes. You can see on the [wiki](https://wiki.cask.co/display/CE/CDAP+Scripts) that we're using compound commands, versus underscored ones. This was something that he specifically had me change in the other implementations, so I think we'll need to apply that here, too.

Maybe this should be `cdap setup hbase`?

```bash
  setup) shift; cdap_setup ${@}; __ret=${?} ;;
```
Then, the `cdap_setup` function would be something like:
```bash
cdap_setup() {
  local __action=${1}
  shift
  cdap_setup_${__action} ${@}
  return $?
}
```